instrumentation/compile-stub/src/main/java/com/ibm/ws/kernel/productinfo/ProductInfo.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Splunk Inc.
```

Comment:
I don't think we can put Splunk copyright here, if you have copied a class verbatim. We have to honour the original license.

Reply:
this class is not copied. it only has methods that are needed by integration with the same signature as origin class has, but method bodies just throw UnsupportedOperationException instead of doing what they do in original class. if you have a better idea for what the copyright should be let me know.

Reply:
What was the original license?

Reply:
https://github.com/OpenLiberty/open-liberty/blob/master/dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/productinfo/ProductInfo.java

Reply:
Ok, they are indeed very different.